packages/react/src/contents/hooks/types/useCommercePages.types.ts

```diff
@@ -0,0 +1,11 @@
+import type { Config, QueryCommercePages } from '@farfetch/blackout-client';
+
+export interface UseCommercePageOptions extends QueryCommercePages {
+  enableAutoFetch?: boolean;
+  strategy?: string;
+  fetchConfig?: Config;
+}
+
+export enum Strategy {
```

Comment:
I think it would be best to rename this to something like `CommercePagesStrategy` as `Strategy` is too general. What do you think?